deploy/Chart/templates/controller/configmaps.yaml

```diff
@@ -12,21 +12,13 @@ data:
       host: "0.0.0.0"
       port: 9443
 
-    storageProvider:
-      provider: "apiserver"
-      apiserver:
-        context: ""
-        namespace: "radius-system"
-
+    storageProvider: {{ toYaml .Values.global.storageProvider | nindent 6 }}
```

Comment:
Is the reason why we start with **storageProvider** and **queueProvider** that they are prerequisites for a Radius installation? We also have **metricsProvider** in the config but metrics service is not required for Radius to stand up. Is that the reason why we started with **storageProvider** and **queueProvider**? Could you give some more information on that?

Reply:
storageProvider and queueProvider both use CRDs which requires cluster-scoped permissions. This allows users to opt out of those definitions if they want.

Reply:
In this iteration, let's not make all provider values configurable. To achieve your goal, it is enough to change only namespace property like below. Let's keep it simple. If you want to make entire object configurable, Please create separate PR later once this pr is merged.

```yaml
      provider: "apiserver"
      apiserver:
        context: ""
        namespace: {{ .Release.Namespace }}
```